components/steps/w-step.vue

```diff
@@ -54,45 +54,42 @@ const getAriaLabel = (props) => {
 const stepClasses = computed(() => [
   ccStep.step,
   {
-    [ccStep.stepVertical]: vertical.value,
-    [ccStep.stepVerticalLeft]: vertical.value && left.value,
-    [ccStep.stepVerticalRight]: vertical.value && !left.value,
-    [ccStep.stepHorizontal]: !vertical.value,
+    [ccStep.vertical]: vertical.value,
+    [ccStep.alignLeft]: vertical.value && left.value,
+    [ccStep.alignRight]: vertical.value && !left.value,
+    [ccStep.horizontal]: !vertical.value,
   },
 ]);
 
-const horizontalClasses = computed(() => [
+const lineHorizontalClasses = computed(() => [
   ccStep.stepLine,
-  ccStep.stepLineHorizontalLeft,
+  ccStep.lineHorizontalAlignLeft,
   {
-    [ccStep.stepLineHorizontal]: !vertical.value,
-    [ccStep.stepLineIncomplete]: !props.active && !props.complete,
-    [ccStep.stepLineComplete]: props.active || props.complete,
+    [ccStep.lineHorizontal]: !vertical.value,
```

Comment:
No need to check for `!vertical.value` here as it is only used on a div with v-if with the same condition, right? 🙂